code/game/atoms.dm

```diff
@@ -616,7 +616,17 @@ var/list/blood_splatter_icons = list()
 /atom/proc/speech_bubble(var/bubble_state = "",var/bubble_loc = src, var/list/bubble_recipients = list())
 	return
 
-/atom/on_varedit(modified_var)
+/atom/vv_edit_var(var_name, var_value)
 	if(!Debug2)
 		admin_spawned = TRUE
-	..()
+	. = ..()
+	switch(var_name)
+		if("light_power", "light_range", "light_color")
+			update_light()
+
+
+/atom/vv_get_dropdown()
+	. = ..()

```

Comment:
Not the jump to turf one? That could be potentially useful for finding the location of a really whackily messed up atom.